openmetadata-spec/src/main/resources/json/schema/entity/services/connections/metadata/atlasConnection.json

```diff
@@ -39,22 +39,10 @@
       "description": "entity types of the data source.",
       "type": "string"
     },
-    "serviceType": {
-      "description": "service type of the data source.",
-      "type": "string"
-    },
     "atlasHost": {
```

Comment:
I see we have `hostPort` which refers to host port of data source is actually not being used anywhere, can you verify that and remove if required, (also if we are removing this lets rename `atlasHost` as `hostPort`)